ITC205_BaseLine_Code/src/FixBookUI.java

```diff
@@ -1,24 +1,23 @@
 import java.util.Scanner;
 
-
 public class FixBookUI {

```

Comment:
D2 - Insert an extra line of whitespace between public class FixbookUI and the import statement

**2 line spaces preceding class declarations